### PR TITLE
Add option to use dark theme

### PR DESCRIPTION
**Pull request by rafaelgomesxyz**

### CODE REVIEW

**Review by MrMamen — DISMISSED** [review]
Nice! You're on a roll!
If I were to give some additional tips, it would be to have a lighter theme for the pop-up-window in light mode.
Also a "follow system default" could be a nice addition.

File: src/components/ThemeWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+		value: 'system',
+		palette: 'light',
+	});
+	const prefersLightMode = useMediaQuery('(prefers-color-scheme: light)', { noSsr: true });
+	const systemPalette = prefersLightMode ? 'light' : 'dark';
+
+	React.useEffect(() => {
+		const setTheme = async () => {
+			const { options } = await BrowserStorage.get('options');
+			const themeValue = options?.theme ?? 'light';
```
Comment (MrMamen) [inline src/components/ThemeWrapper.tsx]: perhaps system should be default?

**Review by MrMamen — DISMISSED** [review]
Visited links are slightly difficult to read on dark theme, but not a major issue.

File: src/components/ThemeWrapper.tsx (hunk 1 above)
Comment (rafaelgomesxyz) [reply src/components/ThemeWrapper.tsx]: Oops, it was supposed to be. Will fix.

**Comment by rafaelgomesxyz** [comment]
> Visited links are slightly difficult to read on dark theme, but not a major issue.

Probably because I was using an anchor instead of a Link component. It should be a bit better now.

**Review by MrMamen — APPROVED** [review]
Well done. I think this is ready for merging now.

**Pull request merged**